src/PowerShellEditorServices/Debugging/DebugService.cs

```diff
@@ -58,6 +58,20 @@ public DebugService(PowerShellContext powerShellContext)
         #region Public Methods
 
         /// <summary>
+        /// Disables all breakpoints in the runspace.
+        /// </summary>
+        /// <returns></returns>
+        public async Task DisableAllBreakpoints()
```

Comment:
No I haven't seen that.  The problem is the ordering of the messages.  Right now we get the launchRequest, setBreakpoints and configurationDone.  But at one point we were getting the launchRequest at the end of this sequence (which is I the code for those two messages tolerates order).  If the launchRequest ever comes after the setBreakpoints then it is too late to "not set" them.  

I guess my confidence in the stability of the debug protocol message ordering isn't extremely high.  :-)  But at the moment, the given order I see on the insiders build suggests that we could just "not set" them.